### ISSUE TITLE
Add new config unit test

### ISSUE THREAD

**Issue by streeve** [ORNL/HydraGNN#51]
Compare CI JSON entries (run directly in CI) to example JSON

**Comment by streeve** [comment]
Bug fixed in #44

**Comment by pzhanggit** [comment]
You made it! Thank you! @streeve

**Comment by allaffa** [comment]
@streeve @pzhanggit 
can we close this issue?

**Comment by streeve** [comment]
No, this is intended to make sure that the input files for untested examples (e.g. LSMS) at least contain everything needed.

**Comment by pzhanggit** [comment]
> 

Current https://github.com/ORNL/HydraGNN/blob/5931f045210e2e873c1e19427ef9ede3ba9e84b3/tests/test_config.py#L18 is for this purpose?

**Comment by streeve** [comment]
Yes, but only currently only checks a few variables
